### ISSUE TITLE
Mender-Walls

### ISSUE THREAD

**Issue by CeartYT** [Anuken/Mindustry-Suggestions#1826]
**Describe the content or mechanics you are proposing.**
-The feature I'm suggesting is called Mender Walls. These are walls that repair themselves and don't need a mender.

**Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.**

-im too lazy to describe how i think this content will improve the game.

**Before making this issue, place an `X` in the boxes below to confirm that you have acknowledged them.** *Failure to do so may result in your request being closed automatically.*

1. - [X] I have done a quick search in the list of suggestions to make sure this has not been suggested yet.
2. - [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
3. - [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
4. - [X] I have read `README.md` to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by DeltaNedas** [comment]
theres about 10 mods that already have this very original concept

**Comment by CeartYT** [comment]
yeah but vanilla mindustry needs this feature, too

**Comment by itcannotbe** [comment]
"im too lazy to describe how i think this content will improve the game."

Another low effort suggestion.

In terms of game balance, with this suggestions has no regard for.
How will this block or basically group of blocks mantain game balance.

And I guess this suggestion entail more wall sprites.

**Comment by BlueWolf3682** [comment]
mods.

**Comment by Yell245** [comment]
Why

**Comment by Kyllingene** [comment]
> -im too lazy to describe how i think this content will improve the game.

Let's say this is perfectly fine. Well, let's just see the sorts of suggestions I could make by using that excuse:


---

> **Describe the content or mechanics you are proposing.**
> A turret that shoots blocks.
> 
> **Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.**
> 
> -im too lazy to describe how i think this content will improve the game.
> 
> **Before making this issue, place an `X` in the boxes below to confirm that you have acknowledged them.** _Failure to do so may result in your request being closed automatically._
> 
> 1. * [x]  I have done a quick search in the list of suggestions to make sure this has not been suggested yet.
> 2. * [x]  I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
> 3. * [x]  I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
> 4. * [x]  I have read `README.md` to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

---

> **Describe the content or mechanics you are proposing.**
> A new item called supermetal that makes walls with 100000 health.
> 
> **Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.**
> 
> -im too lazy to describe how i think this content will improve the game.
> 
> **Before making this issue, place an `X` in the boxes below to confirm that you have acknowledged them.** _Failure to do so may result in your request being closed automatically._
> 
> 1. * [x]  I have done a quick search in the list of suggestions to make sure this has not been suggested yet.
> 2. * [x]  I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
> 3. * [x]  I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
> 4. * [x]  I have read `README.md` to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

---
This might be a little overkill, but you get the idea. The only things wrong with those ideas is that the fake OP never stopped to think through what would happen if they were implemented. To clarify, I do not believe that your suggestion is anywhere as bad as the above examples, I was just painting a picture of what lack of thought can do.

### TL;DR: Think about the consequences of your suggestions and possible alternatives before commenting. It will save you and everyone else (think about Anuke) from wasting time.

**Comment by GreyMario** [comment]
> -im too lazy to describe how i think this content will improve the game.

You're too lazy to design to accomodate Menders, which is incredibly easy to do regardless.

**Comment by rdragonrydr** [comment]
To be fair, there are some times where I'd like to emplace a wall, but there isn't enough room with my cannon placement for it to all fit, or the menders are too far away (yes, even the extended-range ones).

It's rare, though, and it's probably a more satisfying challenge (at least until I get overrun because the solution wasn't good...) to go without it, however.

That said, we already have a lot of bizarre wall types so adding another couldn't hurt too much. It's just that this seems a bit OP. Perhaps if we had to feed the walls a lot of liquid slag to "solidify and seal the cracks"?

It might be fun to exchange the simple "needs more power" for needing to create and pipe in a resource to the walls instead. Especially since Slag is rather power-hungry to create and you need a LOT of mining to create much of the liquid...

**Comment by WinterCrystal** [comment]
What a way to make menders and mend projectors obsolete and throw game balancing out of the window, also, there's a mod that has the block that you wanted.

**Comment by WinterCrystal** [comment]
> yeah but vanilla mindustry needs this feature, too

Please tell me how a block that repairs itself is needed when you can just spam menders or mend projectors behind walls, which can be provided with phase to boost it's efficiency at repairing blocks? Never mind the fact that you can boost an overdrive projector that's within range of a lot of menders/mend projectors and get the same result.

**Comment by itcannotbe** [comment]
Congrats, you just awaken a thread that no replied to in 16 days by a comment that doesn't bring new insights.

Really useful.

**Comment by BlueWolf3682** [comment]
dingus
